<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemCaptionButtons</class>
 <widget class="QWidget" name="SystemCaptionButtons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>43</width>
    <height>16</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QPushButton" name="minimizeButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>13</width>
       <height>12</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>13</width>
       <height>12</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Minimize</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
image: url(:/Graphics/Raster/MinimizeIcon.png);
font-size: 8px;
border-radius: 0.15px;
border-top: 2px solid qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 200));
border-left: 2px solid qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 180));
border-bottom: 3px solid qlineargradient(spread:pad, x1:1, y1:0.994, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 0));
border-right: 2px solid qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 100));
}

QPushButton:pressed {
border-top: 1px solid rgba(0, 0, 0, 128);
border-left: 1px solid black;
border-bottom: 1px solid black;
border-right: 1px solid rgba(0, 0, 0, 128);
padding-left: 1px;
padding-top: 1px
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="miniPlayerButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>13</width>
       <height>12</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>13</width>
       <height>12</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Mini Player</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
image: url(:/Graphics/Raster/MiniPlayerIcon.png);
font-size: 8px;
border-radius: 0.15px;
border-top: 2px solid qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 200));
border-left: 2px solid qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 180));
border-bottom: 3px solid qlineargradient(spread:pad, x1:1, y1:0.994, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 0));
border-right: 2px solid qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 100));
}

QPushButton:pressed {
border-top: 1px solid rgba(0, 0, 0, 128);
border-left: 1px solid black;
border-bottom: 1px solid black;
border-right: 1px solid rgba(0, 0, 0, 128);
padding-left: 1px;
padding-top: 1px
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="closeButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>13</width>
       <height>12</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>13</width>
       <height>12</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Close</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
image: url(:/Graphics/Raster/CloseIcon.png);
font-size: 8px;
border-radius: 0.15px;
border-top: 2px solid qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 200));
border-left: 2px solid qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 180));
border-bottom: 3px solid qlineargradient(spread:pad, x1:1, y1:0.994, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 0));
border-right: 2px solid qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 100));
}

QPushButton:pressed {
border-top: 1px solid rgba(0, 0, 0, 128);
border-left: 1px solid black;
border-bottom: 1px solid black;
border-right: 1px solid rgba(0, 0, 0, 128);
padding-left: 1px;
padding-top: 1px
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
